docs/how-to-guide/how-to-guide.md

```diff
@@ -0,0 +1,358 @@
+# How-to guide
```

Comment:
So far we've been splitting each section in its own file. Is there any reason  why we have changed the approach?

Reply:
I've missed that, let me split up the guides to each on its own section. Thanks for the comment!